docs/projects.md

```diff
@@ -91,7 +89,7 @@ Array of strings that will be passed to the `npx react-native run-ios` command w
 
 #### project.ios.unstable_reactLegacyComponentNames
 
-> Note: Only applicable when new architecture is turned on.
+> Note: Deprecated in React Native 0.74. Only applicable when new architecture is turned on.
```

Comment:
Is there a PR that adds this to iOS as well? Currently https://github.com/facebook/react-native/pull/42294 refers to Android only